src/components/FilterPanel/FilterAccordion/FilterAccordion.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file Filter accordion component.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Accordion } from '../../Accordion';
+import { getComponentNamespace } from '../../../globals/namespace';
+import FilterGroup from '../FilterGroup';
+
+const namespace = getComponentNamespace('filter-accordion');
+
+const FilterAccordion = ({
+  children,
+  className,
+  title,
+  count,
+  countLabel,
+  ...other
+}) => (
+  <FilterGroup
+    className={classnames(namespace, className)}
+    titleClassName={`${namespace}__title`}
+    title={title}
+    count={count}
+    countLabel={countLabel}
+  >
+    <Accordion {...other}>{children}</Accordion>
```

Comment:
@diego-codes I was thinking more that the `{...other}` would go on the parent/root.

Reply:
So I was thinking that since the `FilterAccordion` is a wrapper around the `Accordion` component, it would be more useful to expose the `Accordion` to consumers instead. What do you think?

Reply:
Good point 👍